src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_permute.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/ngraph.hpp"
+#include "./snippets_helpers.hpp"
```

Comment:
Maybe this should work?
```suggestion
#include "snippets_helpers.hpp"
```

Reply:
It most likely would) Changed